front/migrations/db/migration_136.sql

```diff
@@ -0,0 +1,26 @@
+-- -- This migration is dependant on a backfill script
+-- -- The backfill script is: 
+-- -- run psql with --set=backfilled=1 argument if you have rune the script.
+
+CREATE OR REPLACE FUNCTION perform_migration(backfilled boolean DEFAULT false)
+RETURNS VARCHAR AS $$
+BEGIN
+    IF NOT backfilled THEN
+        RAISE NOTICE 'The backfill script:  is required before applying this migation. If you already did it, run psql with --set=backfilled=1 argument.';
+    END IF;
```

Comment:
what backfill is needed for that ?

Reply:
Removed, for some reason our migration generator adds all that stuff (I never want it personally)